src/test/scala/is/hail/TestUtils.scala

```diff
@@ -3,7 +3,7 @@ package is.hail
 import breeze.linalg.{DenseMatrix, Matrix, Vector}
 import is.hail.keytable.KeyTable
 import is.hail.utils._
-import is.hail.variant.{Genotype, VariantDataset}
+import is.hail.variant.{Genotype, VariantDataset, VariantSampleMatrix}
```

Comment:
did you also want to take these out of the import statements?

Reply:
Should be fixed everywhere.